File: Fundamentals/Functions - Exercise/loading_bar.py
def loading_bar(number: int):
    symbols = (number // 10)
    result = f'{number}% [{symbols* "%"}{(10-symbols)*"."}]'
    if symbols < 10:
        print(result)
        return ('Still loading...')
    else:
        print(f'{number}% Complete!')
        return f'[{symbols * "%"}]'


current_number = int(input())
print(loading_bar(number=current_number))

File: JS Fundamentals/Functions - Exercise/loading_bar.js
function loadingBar(numb) {
    if (numb === 100) {
        console.log(`${numb}% Complete!`);
        console.log('[%%%%%%%%%%]');
    } else {
        let bar = [];
        for (let i = 0; i < 10; i++) {
            if (i < numb / 10) {
                bar.push('%');
            } else {
                bar.push('.');
            }
        }
        console.log(`${numb}% [${bar.join('')}]`)
        console.log('Still loading...')
    }
}

loadingBar(30);
loadingBar(50);
loadingBar(100);